packages/genprint/genprint.0.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+version: "0.1"
+synopsis: "PPX syntax extension and library package for printing values of any type"
+description: """
+A PPX syntax extension and library package enabling printing of values of any type using OCaml's internal printing facilities a la toplevel evaluation.
+Useful for debugging as a quick alternative to ocamldebug/ppx_deriving/#install_printer.
+"""
+
+maintainer: "progman1"
+authors: [ "progman1" ]
+homepage: "https://github.com/progman1/genprintlib"
+bug-reports: "https://github.com/progman1/genprintlib/issues"
+dev-repo: "git+https://github.com/progman1/genprintlib.git"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {build}
+  "dune"
+  "ppxlib"
+]
+
+url {
+  src: "https://github.com/progman1/genprintlib/archive/0.1.tar.gz"
+  checksum: "4c7518439c1c4a4d6679190661e84bd0b6b21391"
```

Comment:
This is not a sha256 or and md5. It should be `"md5=176d76e7b541d535f1aee7e194971d0b"` and/or `"sha256=6318065feb4ac2d5000d30e1fced95c9f2987185f7d889ad8c710d03b4765a47"`. How did you get that value?